monai/networks/utils.py

```diff
@@ -281,14 +281,16 @@ def pixelshuffle(
             f"divisible by scale_factor ** dimensions ({factor}**{dim}={scale_divisor})."
         )
 
-    org_channels = channels // scale_divisor
+    org_channels = int(channels // scale_divisor)
```

Comment:
thanks, could you help add a test case like this one https://github.com/Project-MONAI/MONAI/blob/1516ca758090dd8ed2890b64554a77f681a0e224/tests/test_se_blocks.py#L68-L72 for the subpixel upsampling https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_subpixel_upsample.py

Reply:
Done @wyli 👌